site/components/map/ServicesMap.tsx

```diff
@@ -176,6 +176,8 @@ const Map = ({
                         ? { stopTypes: ["MET", "PLT"] }
                         : vehicleMode === Modes.ferry || vehicleMode === VehicleMode.ferryService
                         ? { stopTypes: ["FER", "FBT"] }
+                        : vehicleMode === Modes.rail
```

Comment:
Ticket only mentions adding rail to stop level consequence. Does it need to be included in services level consequences also?

If so these lines of code also need adding to /create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx to allow Rail stations to be selected via the drop down.

Reply:
just checked with steven - just stops consequence page